modules/ROOT/pages/understanding-enterprise-integration-patterns-using-mule.adoc

```diff
@@ -0,0 +1,92 @@
+= Understanding Enterprise Integration Patterns Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Enterprise Integration Patterns are accepted solutions to recurring problems within a given context. The patterns provide a framework for designing, building messaging and integration systems, as well as a common language for teams to use when architecting solutions.
+
+Mule supports most of these patterns shown in the Enterprise Integration Patterns book written by Gregor Hohpe and Bobby Woolf.
```

Comment:
Mule > Mule runtime engine

supports most of these patterns > supports most of the patterns

Reply:
BUT...do we really want to mention this here, since we later list and describe the exact patterns supported?
Consider omitting this. I suspect not all of your audience is familiar with the book cited. You could maybe include it in a See Also bullet?